R/coerce_from_messydate.R

```diff
@@ -7,11 +7,13 @@
 #' into a single date.
 #' For example, `min()`, `max()`, `mean()`, `median()`,
 #' `modal()`, and `random()`.
+#' @return A date object of "Date", "POSIXct" or "POSIXlt" class
 #' @name from_messydate
 NULL
 #> NULL
 
 #' @rdname from_messydate
+#' @return A date object of "Date" class
```

Comment:
I think if you add `@return` to each item on the help file, then you get a weird bullet list of values. Check, but might be best just to have the main `@return` at the top.